js/packages/cli/src/cli-nft.ts

```diff
@@ -32,10 +32,8 @@ programCommand('mint')
   .option('-um, --use-method <string>', 'Optional: Single, Multiple, or Burn')
   .option('-tum, --total-uses <number>', 'Optional: Allowed Number of Uses')
   .option('-ms, --max-supply <number>', 'Optional: Max Supply')
-  .option(
-    '-nvc, --no-verify-creators',
-    'Optional: Disable Verification of Creators',
-  )
+  .option('-nvc, --no-verify-creators', 'Optional: Disable Verification of Creators',)
+  .option('-mm, --mint-multiple <number>', 'Optional: Mint Multiple Tokens',)
```

Comment:
Wondering if this should be just --multiple as mint is already in the name

like 

```
mint --multiple
```